webapi/_test_data/ontologies/anything-onto.ttl

```diff
@@ -419,6 +418,35 @@
 
 
 
+:hasBlueThing rdf:type owl:ObjectProperty ;
+
+               rdfs:subPropertyOf :hasOtherThing ;
+
+               rdfs:label "Ein blaues Ding"@de ,
+                          "Une chose bleue"@fr ,
+                          "Una cosa azzurra"@it ,
+                          "A blue thing"@en ;
+
+               knora-base:subjectClassConstraint :Thing ;
+
+               knora-base:objectClassConstraint :BlueThing ;
+
```

Comment:
`salsah-gui:guiElement salsah-gui:Searchbox ;` has to be added here since you removed  property predicate inheritance. Also permissions are missing.